packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -92,28 +92,33 @@ class PlatformViewsService {
   /// {@template flutter.services.PlatformViewsService.initAndroidView}
   /// Creates a controller for a new Android view.
   ///
-  /// `id` is an unused unique identifier generated with [platformViewsRegistry].
+  /// `id` is an unused unique identifier generated with
```

Comment:
Here and elsewhere in this file: According to style guide sentences should start grammatically correct with a capital letter. May have to reword the sentence to not start with the parameter name.

Reply:
(I realize this is an existing problem, so feel free to punt given the size of this change)